src/components/Comments/Comment.less

```diff
@@ -11,6 +11,14 @@
   }
 }
 
+.Comments > .Comment {
+    overflow:hidden;
+}
+
+.Comments > .Comment > .Comment__text {
+    width:calc(~'100% - 52px');
```

Comment:
this seems to work, but I'm not too sure if hardcoded values would be right.

Reply:
It is `52px` because the zero level comment has `.Avatar` set at `40px` and `.Comment__text` has `margin-left:12px;` , together making 52. I will look on how to not be hardcoded.

Reply:
thanks! great fix though :)